insecure/corruptible/network_common_test.go

```diff
@@ -0,0 +1,147 @@
+package corruptible
+
+import (
+	"fmt"
+	"github.com/onflow/flow-go/insecure"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/model/libp2p/message"
+	"github.com/onflow/flow-go/network/channels"
+	"github.com/onflow/flow-go/network/mocknetwork"
+	"github.com/onflow/flow-go/utils/unittest"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"math/rand"
+	"sync"
+	"testing"
+	"time"
+)
+
+// This test file covers corruptible network tests that are not ingress or egress specific, including error conditions.
```

Comment:
moved comment https://github.com/onflow/flow-go/pull/2811/commits/5f74b06975e10a35416c01a56f93901d5c832c26

Couldn't rename package of test helpers (`insecure/corruptible/network_test_helpers.go`) to `corruptible_test` because it's not a `*_test.go` file.